backend/api/submissions/types.py

```diff
@@ -33,6 +39,7 @@ class Submission:
     duration: "Duration"
     audience_level: "AudienceLevel"
     languages: List["Language"]
+    tags: Optional[List[SubmissionTag]]
```

Comment:
```suggestion
    tags: List[SubmissionTag]
```

The list will be empty by default no? No need to make it optional :)